python/py-pothos/Portfile

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:filetype=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.1
+PortGroup           cxx11 1.1
+PortGroup           python 1.0
+PortGroup           github 1.0
+
+name                py-pothos
+platforms           darwin macosx
+categories-append   science
+license             Boost-1
+maintainers         {@ra1nb0w irh.it:rainbow} {michaelld @michaelld} openmaintainer
+
+description         Python bindings for the Pothos framework
+long_description    ${description}
+
+github.setup        pothosware PothosPython 0.4.0 pothos-python-
+checksums           rmd160  f58ba59802deefa5ea37c8cd2f37bdd6b41687ab \
+                    sha256  fc0fe318c14c3c34d6e63cdad2c080d7d08930fc0d4d75725466ffc1b4a59c0a \
+                    size    28621
+revision            0
+
+python.versions     27 36 37
+python.default_version 36
```

Comment:
Why is this being set to py36?

Reply:
See above. this port is not working and I am asking help :)

Reply:
removed py-pothos for the moment.